en/docs/using-wso2-identity-server/account-Disabling.md

```diff
@@ -0,0 +1,156 @@
+    **Main** tab.
+3.  Expand the **Login Policies** tab.
+4.  Expand the **Account Disabling** tab and select the **Enable Account
+    Disabling** checkbox. Click **Update** to save changes.  
+    ![]( ../../assets/img/103330604/119113135.png) 
+
+    !!! tip
+    
+        Disable the account disabling feature
+    
+        To disable the account disabling option, unselect the **Enable
+        Account Disabling** in each of the tenants that you wish to disable
+        this option for.
+    
+        Alternatively, disable this feature for all tenants by doing the
+        following.
+    
+        1.  Open the `             identity-event.properties            `
+            file found in the
+            `             <IS_HOME>/repository/conf/identity            `
+            folder.
+    
+        2.  Set the following property to **false** to disable account
+            disabling. This will remove the account disabling option from
+            all tenants.
+    
+            ``` java
+                    account.disable.handler.enable=false
+        ```
+
+
+5.  Next, enable account locking. Expand the **Login Policies** tab.
+6.  Expand the **Account Locking** tab and select the **Account Lock
+    Enabled** checkbox. Click **Update** to save changes.  
+    ![]( ../../assets/img/103330604/119113139.png) 
+
+    !!! tip
+    
+        Tip
+    
+        If a user is assigned the **Internal/system** role, the user can
+        bypass account locking even if the user exceeds the specified number
+        of **Maximum Failed Login Attempts**.
+    
+        ![](images/icons/grey_arrow_down.png){.expand-control-image} Click
+        here to see more information
+    
+        WSO2 Identity Server has the **Internal/system** role configured by
+        default. But generally a new user is not assigned the
+        **Internal/syste** m role by default. Required roles can be assigned
+        to a user depending on the set of permission a user needs to have.
+        For more information on roles and permission, see [Configuring Roles
+        and Permissions](_Configuring_Roles_and_Permissions_).
+    
+        Although the **Internal/system** role is configured by default in
+        WSO2 Identity Server, you can delete the role if necessary. To allow
+        users with the **Internal/system** role to bypass account locking,
+        you need to ensure that the role exists in WSO2 Identity Server.
+    
+
+7.  To enable account locking and disabling for other tenants, log out
+    and repeat the steps given above from [step
+    2](#AccountDisabling-step2) onwards.
+
+### Disable an account
+
+Follow the steps below to disable a user account through the WSO2 IS
+management console.
+
+1.  Navigate to **Main\>Claims\>List** and click on '
+    http://wso2.org/claims '.
+
+2.  Edit the **Account Disabled** claim. See [Editing Claim
+    Mapping](_Editing_Claim_Mapping_) for more information on how to do
+    this.
```

Comment:
Links and image paths that were already corrected have been reverted back. Please check in all the relevant places as there are many.

Reply:
Fixed.